RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -22,20 +22,7 @@
     layerList = RecoPixelVertexing.PixelTriplets.quadrupletseedmerging_cff.PixelSeedMergerQuadruplets.layerList.value()
 )
 eras.trackingPhase2PU140.toModify(detachedQuadStepSeedLayers, 
-##FIXME::ERICA layerList need to be moved to PixelLayerTriplets.layerList when the validation to era is done 
-#    layerList = RecoTracker.TkSeedingLayers.PixelLayerTriplets_cfi.PixelLayerTriplets.layerList.value()
-    layerList = ['BPix1+BPix2+BPix3', 'BPix2+BPix3+BPix4',
-                  'BPix2+BPix3+FPix1_pos', 'BPix2+BPix3+FPix1_neg',
-                  'BPix1+BPix2+FPix1_pos', 'BPix1+BPix2+FPix1_neg',
-                  'BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg',
-                  'BPix1+FPix1_pos+FPix2_pos', 'BPix1+FPix1_neg+FPix2_neg',
-                  'FPix1_pos+FPix2_pos+FPix3_pos', 'FPix1_neg+FPix2_neg+FPix3_neg',
-                  'FPix2_pos+FPix3_pos+FPix4_pos', 'FPix2_neg+FPix3_neg+FPix4_neg',
-                  'FPix3_pos+FPix4_pos+FPix5_pos', 'FPix3_neg+FPix4_neg+FPix5_neg',
-                  'FPix4_pos+FPix5_pos+FPix6_pos', 'FPix4_neg+FPix5_neg+FPix6_neg',
-                  'FPix5_pos+FPix6_pos+FPix7_pos', 'FPix5_neg+FPix6_neg+FPix7_neg',
-                  'FPix6_pos+FPix7_pos+FPix8_pos', 'FPix6_neg+FPix7_neg+FPix8_neg',
-                  'FPix6_pos+FPix7_pos+FPix9_pos', 'FPix6_neg+FPix7_neg+FPix9_neg']
+    layerList = RecoTracker.TkSeedingLayers.PixelLayerTriplets_cfi.PixelLayerTriplets.layerList.value()
```

Comment:
looks like the net effect of this change is an addition of 
```
'BPix1+FPix2_pos+FPix3_pos', 'BPix1+FPix2_neg+FPix3_neg',
```

In the Baseline_tilted_200 layout http://mersi.web.cern.ch/mersi/layouts/.current/Baseline_tilted_200/img029.png
it looks like B1-F2-F3 is a fairly unlikely combination